Realm/RLMUtil.mm

```diff
@@ -238,3 +237,18 @@ id RLMValidatedObjectForProperty(id obj, RLMProperty *prop, RLMSchema *schema) {
     }
     return outArray;
 };
+
+NSException *RLMException(NSString *reason, NSDictionary *userInfo) {
+    NSMutableDictionary *info = [NSMutableDictionary dictionaryWithDictionary:userInfo];
```

Comment:
There are currently no tests for the functionality in `RLMUtil`, should I add a new test file?